source/includes/_payment.md

```diff
@@ -3239,41 +2115,16 @@ paymentApi.create_payment_custom_fields(payment_id,
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 201 Created
-< Location: http://127.0.0.1:8080/1.0/kb/payments/8fe697d4-2c25-482c-aa45-f6cd5a48186d/
-< Content-Type: application/json
-< Content-Length: 0
-```
-
-```java
-//First element of the list
-class CustomField {
-    org.killbill.billing.client.model.gen.CustomField@c7d0c38a
-    customFieldId: null
-    objectId: cca08349-8b26-41c7-bfcc-2e3cf70a0f28
-    objectType: PAYMENT
-    name: Test Custom Field
-    value: test_value
-    auditLogs: []
+```json
+{
+  "customFieldId":"7fb3dde7-0911-4477-99e3-69d142509bb9",
+  "objectId":"4927c1a2-3959-4f71-98e7-ce3ba19c92ac",
+  "objectType":"PAYMENT",
+  "name":"Test Custom Field",
+  "value":"test_value",
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?